dorado/scheduling/mission.py

```diff
@@ -164,6 +164,13 @@ def overhead(self, *args, **kwargs):
   elements for VELA 1, a historically important gamma-ray satellite with a
   similar orbit.
 
+* Updated TLE: SSL has now generated an orbital simulation for UVEX. A
+  snapshot TLE is available in the data directory. Note that the torques
```

Comment:
Which data directory? Not the data directory in this repository.

Reply:
It was in the first commit. dorado/schedluing/data/uvex_tle_300_290_i30_72h.tle

Reply:
I see. Well, all the TLEs are in the data directory, so this statement is a bit tautological.

Reply:
Two requests:

1. Reword this to say simply, "The orbit is a snapshot TLE generated by an orbital simulation from SSL" or something similar. No need to mention the data directory.
2. If we are ready to fully replace the toy VELA orbit, then just delete the TLE and delete the bullet point that mentions it.